boards/FSM/Inc/App/App_FsmWorld.h

```diff
@@ -58,16 +60,32 @@ struct HeartbeatMonitor *
  * @param world The world to get primary flow meter for
  * @return The primary flow meter for the given world
  */
-struct FlowMeter *
-    App_FsmWorld_GetPrimaryFlowMeter(const struct FsmWorld *world);
+struct InRangeCheck *
+    App_FsmWorld_GetPrimaryFlowRateInRangeCheck(const struct FsmWorld *world);
```

Comment:
comments in this file don't reflect the fact that flow meter was changed to flow meter in range check

Reply:
Fixed

Reply:
@param and @return still aren't updated to reflect the changes